bundle-workflow/src/test_workflow/perf_test_suite.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+
+
+class PerformanceTestSuite:
+    def __init__(self, bundle_manifest, endpoint, security):
+        self.manifest = bundle_manifest
+        self.work_dir = 'tools/cdk/mensor/mensor_tests'
+        self.endpoint = endpoint
+        self.security = security
+
+    def execute(self):
+        os.chdir(self.work_dir)
+        dir = os.getcwd()
+        # Install the depedencies for the private repo
+        subprocess.check_call('pip3 install boto3 requests setuptools retry dataclasses_json', cwd=dir, shell=True)
```

Comment:
Can we move this into the repo itself / wrap up in a `test.sh`? At least a TODO here.